### ISSUE TITLE
Landscape Issue for Parallel Bible

### ISSUE THREAD

**Issue by VineethaVi** [Bridgeconn/VachanOnline-v2#307]
Description:- Mobile browser:On selecting parallel bible, if we switch to landscape view,only the language drop down is visible.It does not show the bible books or chapters.

steps :- selected the book John 1 in Hindi IRV.
            loaded to the page.
           selected the icon parallel bible from right side bar.
           opened the parallel bible, language drop down not visible.

![IMG_20230511_090611](https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/e71931e9-496d-4f8a-a3a3-864ec5451054)

Expecting Result : when opened parallel Bible in landscape mode, show the drop-down of language and Bible books .

**Comment by RevantCI** [comment]
As this is also the mobile browser and there is not enough space to show all the dropdowns we have made it the same as the browser view portrait view. 
The landscape view is not very usable as there is a lack of space but for now all the buttons should be visible.

**Comment by AnnCherianA** [comment]
[like]  Amreeta Ann Cherian reacted to your message:
________________________________

**Comment by RevantCI** [comment]
![image](https://github.com/Bridgeconn/VachanOnline-v2/assets/12570574/8fc2dd0c-0f21-4fbe-b133-7776ba188359)

**Comment by RevantCI** [comment]
The screenshot I added here https://github.com/Bridgeconn/VachanOnline-v2/issues/307#issuecomment-1543689314 is after fixing the issue
I have explained the fix we have done here - https://github.com/Bridgeconn/VachanOnline-v2/issues/307#issuecomment-1543687949

**Comment by AnnCherianA** [comment]
@VineethaVi please verify this issue and share the result

**Comment by VineethaVi** [comment]
It is fix. I verified it and it is working fine.
steps- selected the book John
selected the parallel bible from the bottom bar in landscape mode.
parallel bible opened.
I can see drop down of language  and books.
I can't see the words (due to the screen size of the phone.)

![IMG_20230511_170929](https://github.com/Bridgeconn/VachanOnline-v2/assets/132653333/55039f9a-e48d-4d8d-b72c-934840db908a)
It is working fine and moving to QA review.

**Comment by JesniJ** [comment]
Test results are verified and those were documented clear with screenshots , therefore moving this issue to done.

**Comment by RevantCI** [comment]
Sure for landscape view some things may not look nice in parallel view, we have to see it case by case for each panel and see how to show it. Sure we can raise an issue about it separately to study the views and change it as required.

**Comment by AnnCherianA** [comment]
Moving to QA 
@RevantCI thanks for confirming
@VineethaVi  please move this issue to Done  directly and open a new issue for studying the behaviour of different features in  landscape view .

**Comment by VineethaVi** [comment]
moving to Done
